### PR TITLE
FP-294:  Featured resource.

### PR DESCRIPTION
**Pull request by robherba**
# Ticket(s)

- [FP-376:  Featured resource](https://fourkitchens.clickup.com/t/36718269/FP-376)

## Purpose

**This PR does the following:**

- Adds a `featured resource` component.

## Dependencies:

- https://github.com/fourkitchens/forcepoint-nextjs-page-router/pull/55
- https://github.com/fourkitchens/forcepoint-drupal/pull/93

## Functional Testing

- [x] Navigate to`featured resource` page on storybook.
- [x] Card version: http://localhost:6006/?path=/story/components-featured-resource--card

       
Captura de pantalla 2024-04-24 a la(s) 12 18 30 p  m

- [x] List version: http://localhost:6006/?path=/story/components-featured-resource--list

imagen

- [x] Validate that everything looks and works as expected.

## Designs

- [Card version](https://www.figma.com/file/HwazkcYwlIm9Yz6NwIQ1b3/Web-Standards?type=design&node-id=2256-129374&mode=design&t=aDa2TsAcVKMllX9K-0)
- [List version](https://www.figma.com/file/HwazkcYwlIm9Yz6NwIQ1b3/Web-Standards?type=design&node-id=2269-129495&mode=design&t=aDa2TsAcVKMllX9K-0)

### CODE REVIEW

**Comment by fkbender** [comment]
Task linked: [FP-294 Featured Resource - FED](https://app.clickup.com/t/36718269/FP-294)
Task linked: [FP-376 Featured Resource - BED](https://app.clickup.com/t/36718269/FP-376)

**Review by robert-arias — CHANGES_REQUESTED** [review]
Everything looks good toca! Just some observations.

**Comment by robert-arias** [comment]
Also, Should the lists have a text below the title?
image

File: src/lib/components/02-components/featured-resource/featured-resource--card.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  ) : null;
+
+  return (
+    <div className="md:white rounded-m max-md:border-2 max-md:border-azure md:max-w-[480px] md:bg-azure">
+      <div className="overflow-hidden rounded-m max-md:hidden md:max-h-[240px]">
+        {renderedImage}
+      </div>
+      <div className="flex flex-col gap-md p-md md:gap-lg md:p-lg">
+        {title && (
+          <span className="text-h3 font-[600] text-black md:text-h4">
```
[the remark this answers is not in the document]
Comment (robherba) [reply src/lib/components/02-components/featured-resource/featured-resource--card.tsx:46]: Hi Toca, thanks for capturing this. It seems the font size changes depending on the device size (mobile: 24px / desktop: 20px). Please check out this URL and let me know if it makes sense or if you recommend making adjustments: [[**Figma Link**](https://www.figma.com/file/HwazkcYwlIm9Yz6NwIQ1b3/Web-Standards?type=design&node-id=2247-127303&mode=design&t=E5hmSSnYa8iKqFrL-4)]

**Comment by robherba** [comment]
> Also, Shouldn't the lists have a text below the title? image

I had the same question too. In a chat with Marvin and Duncan, we discussed the list version component. The styles for this version are in Link A, while those in Link B are for a different component. Also, I talked to Duncan separately about the icons we should use, and he suggested using the ones from Link C.

[Link A](https://www.figma.com/file/HwazkcYwlIm9Yz6NwIQ1b3/Web-Standards?type=design&node-id=2252-127847&mode=design&t=qw517ZtAiR0CAXvK-4)

[Link B](https://www.figma.com/file/HwazkcYwlIm9Yz6NwIQ1b3/Web-Standards?type=design&node-id=2252-128631&mode=design&t=qw517ZtAiR0CAXvK-4)

[Link C](https://www.figma.com/file/HwazkcYwlIm9Yz6NwIQ1b3/Web-Standards?type=design&node-id=4263-11044&mode=design&t=YCjWfvmsFZqKU0IU-4)

**Pull request merged**